### PR TITLE
Homework3:Anton Gorulev

### PR DESCRIPTION
**Pull request by AntonGorulev**
### PR checklist
Please make sure you've passed PR checklist
- [ ] my PR passes CI checks (flake8 and pycodestyle)
- [ ] my PR named according to guidelines.
- [ ] my PR contains complete tasks descriptions for all tasks
- [ ] my PR made from the fork feature branch to maxim homework# branch.
- [ ] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore

### CODE REVIEW

File: src/Homework2./task1.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'''
```
Comment (belovmd) [inline src/Homework2./task1.py:1]: bad folder name. should be snake_case without special symbols

File: src/Homework3./task1.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+'''
```
Comment (belovmd) [inline src/Homework3./task1.py:1]: bad file name

File: src/Homework3./task1.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+Напишите программу, которая печатает цифры от 1 до 100,
+но вместо чисел, кратных 3 пишет Fizz,
+вместо чисел кратный 5 пишет Buzz,
+а вместо чисел одновременно кратных и 3 и 5 - FizzBuzz
+'''
+
+for x in range(1, 100):
+    if x % 3 == 0 and x % 5 == 0:
```
Comment (belovmd) [inline src/Homework3./task1.py:9]: 15?

File: src/Homework3./task2.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+элемент '2a' так чтобы в исходном списке этого элемента не было.
+'''
+import copy
+
+list_ = [_ + _1 for _ in 'ab' for _1 in 'bcd']
+print(list_)
+
+print(list_[::2])
+
+list_1 = [str(_) + 'a' for _ in range(1, 5)]
```
Comment (belovmd) [inline src/Homework3./task2.py:20]: '1234'?

File: src/Homework3./task3.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+list_ = ['a', 'b', 'c']
+print(tuple(list_))
+
+tuple_ = ('a', 'b', 'c')
+print(list(tuple_))
+
+a, b, c = 'a', 2, 'pytnon'
+print(a, b, c)
+
+tuple_1 = ('1, 2, 3', )
```
Comment (belovmd) [inline src/Homework3./task3.py:19]: doesn't solve task in description

File: src/Homework3./task3.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+for _ in tuple_1:
+    print(_)
```
Comment (belovmd) [inline src/Homework3./task3.py:21]: bad naming. _ - for unused vars

File: src/Homework3./task5.py [hunk 7]
```diff
@@ -0,0 +1,10 @@
+'''
+Дан список. Выведите те его элементы, которые встречаются в списке
+только один раз. Элементы нужно выводить в том порядке,
+в котором они встречаются в списке.
+'''
+list_ = [3, 4, 0, 1, 2, 3, 3, ]
+for _ in list_:
+    list_.count(_)
+    if list_.count(_) == 1:
```
Comment (belovmd) [inline src/Homework3./task5.py:9]: O(n**2) solution. can be solved with O(n)

File: src/Homework3./task5.py [hunk 8]
```diff
@@ -0,0 +1,10 @@
+'''
+Дан список. Выведите те его элементы, которые встречаются в списке
+только один раз. Элементы нужно выводить в том порядке,
+в котором они встречаются в списке.
+'''
+list_ = [3, 4, 0, 1, 2, 3, 3, ]
+for _ in list_:
```
Comment (belovmd) [inline src/Homework3./task5.py:7]: bad naming

File: src/Homework3./task6.py [hunk 9]
```diff
@@ -0,0 +1,14 @@
+'''
+Дан список целых чисел.Требуется переместить все ненулевые
+элементы в левую часть списка, не меняя их порядок,
+а все нули - в правую часть. Порядок ненулевых элементов
+изменять нельзя, дополнительный список использовать нельзя,
+задачу нужно выполнить за один проход по списку.
+Распечатайте полученный список.
+'''
+list_ = [0, 1, 2, 3, 0, 4, ]
+for _ in list_:
```
Comment (belovmd) [inline src/Homework3./task6.py:10]: bad naming

**Review by belovmd — CHANGES_REQUESTED** [review]
sanity checks failed: redundant files in code (hw1, hw2)

please address next issues

**Comment by AntonGorulev** [comment]
Add new PR for Homework3

**Pull request closed without merging**